spec/models/book_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+require 'faker'
+
+describe Book, type: :model do
```

Comment:
`describe Book do`

Reply:
`subject(:book) { build(:book) }` use factory_bot and faker
`it { is_expected.to be_valid }`